### ISSUE TITLE
Docs for synapse-s3-storage-provder erroneously say to not include endpoint for S3

### ISSUE THREAD

**Issue by JPinSPACE** [spantaleev/matrix-docker-ansible-deploy#2330]
I would just make this change myself and submit a pull request but I thought it was important that someone more familiar with the project confirm my findings first.

In [this guide](https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/master/docs/configuring-playbook-synapse-s3-storage-provider.md) at the line `matrix_synapse_ext_synapse_s3_storage_provider_config_endpoint_url:` the user is instructed to "delete this whole line for Amazon S3".

This is incorrect and lead to my media storage having all sorts of issues, including user/room avatars not loading, and required a fair amount of troubleshooting. Just as a hunch, I searched for the S3 endpoint for my region, put the line back with the correct endpoint, re-ran the playbooks and restarted everything.. and my avatars were restored!

tldr: S3 users **do** need to include an endpoint url.

**Comment by spantaleev** [comment]
I suppose you're right - it's probably required. I've updated the docs and validation logic!

**Comment by JPinSPACE** [comment]
Great! Thanks for all your hard work. It's made it possible for me to run this stuff at all.
